modules/ROOT/pages/_partials/data-sync/server-using-mutations-with-a-data-sync-server-and-a-data-sync-client.adoc

```diff
@@ -0,0 +1,122 @@
+[id="server-using-mutations-with-a-data-sync-server-and-a-data-sync-client-{context}"]
+= Using Mutations with a Data Sync Server
+
+Build a Data Sync Server and query the mutation.
+The Data Sync Server will be running locally in this procedure.
+
+.Prerequisites
+
+* link:https://github.com/jstaffor/mobile-docs/blob/master/modules/ROOT/pages/_partials/sync/server-getting-started.adoc[Getting Started with Hello World Data Sync].
+* You are familiar with {sync-server} operations, for example, stopping and starting {sync-server}.
+
+.Procedure
+
+. If the {sync-server} is running, stop it.
++
+. Edit the `index.js` file and replace its contents with the following content:
```

Comment:
If we leave https://mdc-docs.netlify.com/aerogear/latest/ds-query/ in place, we dont' need this

Reply:
I think this is the next phase of this work as per your email where you outlined this structure: https://docs.google.com/spreadsheets/d/1eebcc4IGWnD-0xI4qYTly3pLyXxVujO17121WfOgf1c/edit?usp=sharing

I hope that we can use what is done in this PR to further the next phase :+1: